www/search

```diff
@@ -222,18 +222,7 @@ $qterm = mysql_real_escape_string($term, $db);
 $hrefTerm = ($term ? $term : $tTerm);
 $hrefTerm = urlencode($hrefTerm);
 
-// get the search type
-const SEARCH_TYPES = ['list', 'poll', 'member', 'comp', 'tag'];
-$searchType = 'game';
-$searchButton = "Search Games";
-foreach (SEARCH_TYPES as $st) {
-    if (isset($_REQUEST[$st])) {
-        $searchType = $st;
-        break;
-    }
-}
-
-$browseTabs = [
+const BROWSE_TABS = [
```

Comment:
I feel like this would be better named `SEARCH_TYPES` than `BROWSE_TABS`. In the current implementation, it's mostly used for types of searches.